kiwi/boot/functions.sh

```diff
@@ -5525,18 +5540,25 @@ function waitForStorageDevice {
     # which could be a whole disk or a partition.
     # the function will wait until the size of the
     # storage device could be obtained or the check
-    # counter equals 4
+    # counter equals 30. For USB devices it waits
+    # only until check counter equals 2.
     # ----
     local IFS=$IFS_ORIG
     local device=$1
     local check=0
+    local transport="`getDeviceTransportType $device`"
+    if [[ $transport == *"usb"* ]];then
```

Comment:
I'm not sure how many usb transport type exists, but is it a safe regular expression to match any string that has the three letters 'usb' in its name ? Just asking, maybe a simple string match works for usb devices ?